package.json

```diff
@@ -71,9 +69,13 @@
     "time-grunt": "1.2.1"
   },
   "scripts": {
-    "test": "grunt test; grunt lint",
-    "start": "node --harmony-arrow-functions index.js",
-    "startdev": "iojs --harmony-arrow-functions index.js --development -c"
+    "build": "cp -r lib dist && babel --out-dir=dist index.js 'lib/**/*.js'",
```

Comment:
it does mean people will have to boot from `npm run startdev`, or `npm run build` then node the dist/index.js.  there will be some learning for that.  what do you think is proper?  README.md update and slack broadcast?